### PR TITLE
jmxweb plugin - add support for automated and scheduled monitor repor…

### PR DESCRIPTION
**Pull request by deleolajide**
…ts via email

### CODE REVIEW

**Comment by guusdk** [comment]
log4j, and both slf4j libraries are supplied by openfire. No need to add
them to the plugin.

 - Guus

**Comment by deleolajide** [comment]
Sure thing!. Thanks, I will remove them.

File: src/plugins/jmxweb/src/java/com/ifsoft/jmxweb/plugin/EmailScheduler.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        } catch (SchedulerException se) {
+            Log.error("stopMonitoring", se);
+        }
+    }
+
+    @Override
+    public void execute(JobExecutionContext jobExecutionContext) throws JobExecutionException {
+        Log.info( "Email Monitoring Running");
+        try {
+            /*EmailService.getInstance().sendMessage(user.getName(), user.getEmail(), "Openfire",
```
Comment (jcalderonzumba) [inline src/plugins/jmxweb/src/java/com/ifsoft/jmxweb/plugin/EmailScheduler.java]: Sorry to intrude, i usually watch the evolution of Openfire and all i can say is that having GIT as the VCS means you can safely remove this commented code. No need to push commented code to the repository.

File: src/plugins/jmxweb/src/java/com/ifsoft/jmxweb/plugin/HttpClient.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import java.net.URL;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.lang.*;
+import org.jivesoftware.util.*;
+import org.jivesoftware.openfire.XMPPServer;
+
+public class HttpClient {
+    private static Logger Log = LoggerFactory.getLogger("JmxWebPlugin:HttpClient");
+    // http://localhost:7070/jolokia/read/java.lang:type=Memory/HeapMemoryUsage
```
Comment (jcalderonzumba) [inline src/plugins/jmxweb/src/java/com/ifsoft/jmxweb/plugin/HttpClient.java]: Same here,

**Pull request merged**